### DISCUSSION TITLE
App Icons 3.0: Final icon selection

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Notes#272] [Polls]
This is the poll where we have to select both the shape and the color. No other contenders or modifications are allowed.

The icon with the most numeric votes will be the winner, regardless of how close the votes are compared to each other.

> [!IMPORTANT]
> Target date for end of poll: Sunday, August 4th.

|     | A   | B   |
| --- | --- | --- |
| 1   | ![Final selection_image](https://github.com/user-attachments/assets/f998d17f-35f7-4b87-9b25-99bca467ef63) | ![1_Final selection_image](https://github.com/user-attachments/assets/be4217b8-2f80-4484-a4d3-97544ef03e0a)
| 2   | ![2_Final selection_image](https://github.com/user-attachments/assets/bfe9c101-8b42-42dd-80ef-071464f4536f) | ![3_Final selection_image](https://github.com/user-attachments/assets/26586b97-45af-493b-8118-4d98475d97da) |
| 3   | ![5_Final selection_image](https://github.com/user-attachments/assets/ad9d83e5-c21f-4370-91c8-4f7decaa0f96) | ![4_Final selection_image](https://github.com/user-attachments/assets/17ba32fb-87c9-4300-a199-fef82720bbbc) |
| 4   | ![6_Final selection_image](https://github.com/user-attachments/assets/9af878b0-453c-49c9-b37e-985ba1bd249d) | ![7_Final selection_image](https://github.com/user-attachments/assets/3af82d74-1afd-47bd-a15e-4bee5c5ef16c) |

> [!NOTE]
> The fourth row has some other similar variants, see https://github.com/orgs/TriliumNext/discussions/261#discussioncomment-10126634.

**Comment by meichthys** [comment]
I'm a bit confused:
So if 1A has 5 votes and 1B has 6 votes and 2A has 7 votes, then the icon will be 2A even though 1 had more total votes?

**Reply by eliandoran** [reply]
@meichthys , the icon with the most votes will win, the columns are just to distinguish the color. So the row by itself is irrelevant, all that matters is the vote for every one of the 8 icons.

**Reply by meichthys** [reply]
Got it. Thanks.

**Comment by zerebos** [comment]
I'm more than happy with several of these, but I voted my number 1. My top 3 would be:

1. 3a
2. 2a
3. 4c (shape of 4b but with the original colors)

**Reply by meichthys** [reply]
> 4c (shape of 4b but with the original colors)

That's my preference

**Reply by Alumniminium** [reply]
same

**Reply by meichthys** [reply]
I think If 4a or 4b gets the votes we could make that small adjustment.

**Reply by diyoyo** [reply]
> 3\. 4c (shape of 4b but with the original colors)

Me too

**Comment by TorstenC** [comment]
## **3A (third row, multi-color) - why?**

The proposal for the "3A (third row, multi-color)" icon is the best decision for several compelling reasons. This argument is grounded in established best practices for app icon design, which emphasize simplicity, recognizability, and brand alignment.

![3A (third row, multi-color)](https://github.com/user-attachments/assets/ad9d83e5-c21f-4370-91c8-4f7decaa0f96)

### Simplicity

One of the most critical aspects of effective app icon design is simplicity. According to multiple design experts, a simple icon is easier to recognize and remember, which is crucial in a crowded app marketplace[1][3][5]. The "3A" icon adheres to this principle by using clean lines and a straightforward design that avoids unnecessary details. This simplicity ensures that the icon is not only visually appealing but also functional across various sizes and platforms, maintaining its clarity and impact whether viewed on a small mobile screen or a larger desktop display.

### Recognizability

The "3A" icon's design is not only simple but also highly recognizable. Recognizability is essential for user engagement and retention, as users are more likely to interact with an app whose icon they can easily identify[1][4]. The multi-color scheme of the "3A" icon helps it stand out while still being associated with the known Trilium brand. This balance between uniqueness and familiarity ensures that users can quickly locate the app among others on their device, enhancing the user experience.

### Brand Alignment

An app icon must align with the overall brand identity to strengthen brand recognition and loyalty[1][3][6]. The "3A" icon closely resembles the established Trilium logo, which helps maintain a consistent brand image. This consistency is vital for building trust and credibility with users, as it reinforces the brand's visual identity across different touchpoints. The use of multiple colors in the "3A" icon also reflects the dynamic and versatile nature of the Trilium app, further aligning the icon with the brand's core values and functionalities.

## Technical Considerations

From a technical perspective, the "3A" icon is well-suited for scalability and adaptability. The design's simplicity ensures that it remains legible and visually appealing at various sizes, which is crucial given the different display requirements across devices[3][5]. Additionally, the multi-color approach provides good contrast, making the icon easily distinguishable against various backgrounds, which enhances its visibility and usability[4].

## Conclusion

In summary, the "3A (third row, multi-color)" icon is the best choice for TriliumNext's app icon due to its adherence to best practices in app icon design. Its simplicity, recognizability, and alignment with the Trilium brand make it a standout option that will likely enhance user engagement and brand loyalty. By choosing this icon, TriliumNext can ensure that its app is easily identifiable and appealing to users, thereby supporting its overall success in the competitive app marketplace.

Citations:
[1] https://www.appsflyer.com/blog/tips-strategy/app-icon-design/
[3] https://stfalcon.com/en/blog/post/14-app-icon-design-tips
[4] https://adapty.io/blog/how-to-design-app-icon/
[5] https://www.apptweak.com/en/aso-blog/how-to-design-an-app-icon
[6] https://arounda.agency/blog/how-to-improve-your-app-icon-design

**Reply by Alumniminium** [reply]
ChatGPT is getting really good at citing sources :exploding_head:

**Reply by meichthys** [reply]
I'm pretty sure number four would satisfy all of these criteria as well 😛

**Reply by hoggatt** [reply]
I'm sort of sick of crappy annoying boring corporate logos that have 0 complexity and are just blehhh.

That's why I voted for A1. It feels unique and much more human.

**Comment by adoriandoran** [comment]
I consider 2A to be the ideal icon for TrilliumNext.

Its resemblance to Zadam's Trillium will remind us of the legacy of the original project, while the slightly different silhouette will indicate a new stage in its evolution.

I also believe that TrilliumNext should use an icon with something distinctive from the original Trillium, since it is a fork.

**Comment by eliandoran** [comment]
@TriliumNext/everyone , looking so far at the poll it is clearly noticeable that either 1A or 2A will win the poll. For the people who have voted another icon, please consider revoting either 1A or 2A to clearly distinguish your preference between the two.

This is especially relevant for people who have voted either 1B or 2B, voting their colored counterpart would make your vote count.

**Reply by zerebos** [reply]
I don't think GitHub allows you to change your vote, does it?

**Reply by meichthys** [reply]
yes, click hide results and then choose a different option to vote for.

**Reply by zerebos** [reply]
Ah something is broken on my end then because clicking hide results does nothing 😄

**Comment by Br0kenSilos** [comment]
For me, I like.

1a
2a

**Comment by eliandoran** [comment]
End date has been reached.

... and the winner is:

![2A](https://github.com/user-attachments/assets/bfe9c101-8b42-42dd-80ef-071464f4536f)

**Reply by perfectra1n** [reply]
I believe that the entirety of planet earth requires it to be purple, however. /s

**Reply by eliandoran** [reply]
@perfectra1n , it looks superb on Windows:

![image](https://github.com/user-attachments/assets/1bcd6f50-e666-4d4b-9bb5-072081827038)

No /s needed.

**Reply by zerebos** [reply]
To each their own 😄 I'm not a fan haha

**Reply by Alumniminium** [reply]
Democracy never works, the majority is always wrong :D

**Reply by ashleysommer** [reply]
If 2A is the winner, why does 1A have more votes? Did more people vote for that after the winner was chosen?

**Reply by meichthys** [reply]
That's correct, at the last day of the vote period, 2A had more votes.

**Comment by perfectra1n** [comment]
Would it be possible to let the user select what icon they want?

**Reply by eliandoran** [reply]
@perfectra1n , theoretically yes but I think it's quite a big task due to being multi-platform. You'd also have to change the launch configurations and so on.
